library/src/main/java/com/schibsted/spain/barista/BaristaClickActions.kt

```diff
@@ -43,14 +41,14 @@ object BaristaClickActions {
         try {
             try {
                 clickDisplayedView(viewMatcher, spyHandler)
-            } catch (exception: NoMatchingViewException) {
+            } catch (fistError: RuntimeException) {
```

Comment:
typo! I don't think this kind of error is like a fist 👊 right?

Reply:
Haha should be!